src/actions/Bluetooth/SensorDownloadActions.js

```diff
@@ -129,6 +129,11 @@ const downloadLogsFromSensor = sensor => async dispatch => {
             );
 
             await TemperatureLogManager().saveLogs(temperatureLogs);
+
+            await BleService().updateLogInterval(macAddress, logInterval);
```

Comment:
Possibly worth including some retries here, if it's quite an important thing to have set? Less important for the log clearing probably since we don't really care if that fails now and again.

Reply:
2989b51.